go

```diff
@@ -64,8 +64,13 @@ def update_gems
   exec_cmd 'bundle'
 end
 
+def update_submods
+  exec_cmd "git submodule update --remote"
+end
+
 def update_data
-  exec_cmd 'cd _data && ./import-public.rb'
+  ruby = exec_cmd 'which ruby'
```

Comment:
Heh, neat trick. I'll consider doing that in the Hub's go script to avoid hardcoding the path.